src/main/java/com/openlattice/kindling/search/ConductorElasticsearchImpl.java

```diff
@@ -726,33 +726,6 @@ public boolean createBulkEntityData( UUID entitySetId, Map<UUID, Map<UUID, Set<O
         return true;
     }
 
-    @Override
-    public boolean updateEntityData( EntityDataKey edk, Map<UUID, Set<Object>> propertyValues ) {
```

Comment:
Where did this go / what is happening to this?

Reply:
it wasn't actually used anywhere